### ISSUE TITLE
tibble::add_column - Non-Generic Function Creating Bind Issue

### ISSUE THREAD

**Issue by mdancho84** [business-science/tibbletime#11]
There is an issue with `tibble` `add_column()` causing issues with the binding operation for `tbl_time` class. We are working on it!
```
> test
# A time tibble: 3 x 2
# Index: date
        date    price
*     <date>    <dbl>
1 2017-01-03 55.01466
2 2017-02-01 55.55809
3 2017-03-01 54.23100
> test %>% add_column(dummy = rep(1, 3))
Error: `expr` must quote a symbol, scalar, or call
```

**Comment by DavisVaughan** [comment]
This should now be fixed by https://github.com/tidyverse/tibble/pull/306. Install the dev version of tibble if this feature is required.

Also install `tibbletime (>= 0.0.1.9001)`

**Comment by DavisVaughan** [comment]
Fixed in tibble 1.4.1
